ruka/ingress/lhn/ingress-s3-lhn.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  annotations:
+    cert-manager.io/cluster-issuer: letsencrypt
+    kubernetes.io/ingress.class: nginx-lhn
+  name: rook-ceph-rgw-ingress-lhn
+  namespace: rook-ceph
+spec:
+  rules:
+  - host: s3-lhn.dev.lsst.org
```

Comment:
since this isn't an end-user facing service, how about s3-lhn.ruka.dev.lsst.org?

Reply:
i had it like that, going to change it back 👍🏻